### PR TITLE
do basic validation in the export analyzer

### PR DESCRIPTION
**Pull request by mikehelmick**
Adds basic data validation when printing export file.

```release-note
NONE
```

### CODE REVIEW

File: tools/export-analyzer/main.go [hunk 1]
```diff
@@ -45,9 +49,44 @@ func main() {
...
 	prettyJSON, err := json.MarshalIndent(keyExport, "", "  ")
 	if err != nil {
 		log.Fatalf("error pretty printing export: %v", err)
 	}
 	log.Printf("%v", string(prettyJSON))
 }
+
+func checkExportFile(export *exportpb.TemporaryExposureKeyExport) {
+	now := time.Now().UTC()
+	floor := model.IntervalNumber(now.Add(-14 * 24 * time.Hour))
```
Comment (gurayAlsac) [inline tools/export-analyzer/main.go:66]: nit: could define 14 and 16 (below) as self-describing constants up above?

Comment (mikehelmick) [reply tools/export-analyzer/main.go:66]: Will do in follow up

**Pull request merged**